content/ko/docs/setup/production-environment/container-runtimes.md

```diff
@@ -87,22 +87,67 @@ sudo sysctl --system
 리눅스에서, {{< glossary_tooltip text="control group" term_id="cgroup" >}}은
 프로세스에 할당된 리소스를 제한하는데 사용된다.
 
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}}과
+그에 기반하는 컨테이너 런타임 모두 control group들과 상호작용 해야 하는데, 이는
+[파드와 컨테이너의 자원 관리](/docs/concepts/configuration/manage-resources-containers/)를 하도록 하며
+cpu 혹은 메모리와 같은 자원의 요청과 제한을 설정하기 위함이다. control
+group과 상호작용하기 위해서는, kubelet과 컨테이너 런타임이 *cgroup 드라이버*를 사용해야 한다.
+매우 중요한 점은, kubelet과 컨테이너 런타임이 같은 cgroup 
+group driver를 사용해야 하며 동일하게 구성되어야 한다는 것이다.
```

Comment:
```suggestion
매우 중요한 점은, kubelet과 컨테이너 런타임이 같은 cgroup 
드라이버를 사용해야 하며 구성도 동일해야 한다는 것이다.
```

It's critical that the kubelet and the container runtime uses the same cgroup
driver and are configured the same.